grader-api/src/main/java/org/sourcegrade/jagr/api/testing/CompileResult.java

```diff
@@ -33,4 +33,6 @@
   int getErrorCount();
 
   int getOtherCount();
+
+  RuntimeResources getResourceInfo();
```

Comment:
```suggestion
  RuntimeResources getRuntimeResources();
```

Reply:
This would hide the `runtimeResources` property of `JavaCompiledContainer`. We could rename the `runtimeResources` property of `JavaCompiledContainer` to something else and then rename `getResourceInfo()` to `getRuntimeResources()`.

Reply:
No, it's okay to give them both the same name. It will be necessary to make the constructor property private, but the value is still accessible via the getter.